negotiation/contract.negotiation.binding.https.md

```diff
@@ -61,7 +61,9 @@ the [ContractNegotiation](./message/contract-negotiation.json):
 
 Predefined states are: `REQUESTED`, `OFFERED`, `ACCEPTED`, `AGREED`, `VERIFIED`, `FINALIZED`, and `TERMINATED`.
 
-If the negotiation does not exist or the client is not authorized, the provider connector must return an HTTP 404 (Not Found) response.
+If the client is not authorized, the provider connector must return an HTTP 401 (Unauthorized) response.
+
+If the negotiation does not exist, the provider connector must return an HTTP 404 (Not Found) response.
```

Comment:
This changes the response and opens a potential security hole. HTTP 404 was explicitly used in both cases to avoid providing unnecessary information to malicious clients fishing for identifiers.